### DISCUSSION TITLE
Strategy for dealing with recalcitrant IT

### DISCUSSION THREAD

**Discussion by mprogers** [dotnet/maui#8862] [Q&A] [answered]
Hi,

I work at a university, and they won't install Visual Studio 2022 Preview on lab machines because it's beta. I tried teaching this last year and it just looked too rough: now the product looks almost ready for prime-time, or at least good enough for my purposes, but IT has put the kibosh on that, so I need a plan B.

Does anybody have any ideas as to how I might get around this? Something magical involving docker (which they might be willing to work with me on)? Some online virtual environment that won't cost an arm and a leg (did I mention we're a university)? A sworn affidavit from some developers at Microsoft that it won't cause issues with other software? 😀 

Thanks for any advice/info

Michael

**Comment by marcmognol** [comment, marked as answer]
It should be ok now as MAUI is available in Visual Studio 2022.

**Reply by mprogers** [reply]
That's what I thought too, but they haven't enabled developer mode, so I can't do Windows development. Android development also requires administrative permissions, hopefully they can help me out. I wish I could use an Azure VM, but I can't find anyone at Microsoft who supports higher-ed faculty and can speak about pricing.
